tests/integ/test_dataframe.py

```diff
@@ -152,7 +152,7 @@ def test_dataframe_get_attr(session):
     assert "object has no attribute" in str(exc_info)
 
 
-@pytest.mark.localtest
+# @pytest.mark.localtest
```

Comment:
temporarily skipped because the underlying stage registration is reimplemented.
will reenable in the following file operation get support pr